jenkins-pipelines/longevity-sla-system-24h.jenkinsfile

```diff
@@ -0,0 +1,12 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: 'test-cases/longevity/longevity-sla-system-24h.yaml'
```

Comment:
Please move the test yaml to be under "entrprise-features/sla/longevity/"

Reply:
done